src/plugins/presentation_util/public/components/solution_toolbar/items/primary_popover.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { SolutionToolbarPopover, Props as SolutionToolbarPopoverProps } from './popover';
+
+export type Props = Omit<SolutionToolbarPopoverProps, 'primary'>;
+
+export const PrimaryActionPopover = (props: Omit<SolutionToolbarPopoverProps, 'primary'>) => (
+  <SolutionToolbarPopover primary={true} {...props} />
+);
+
+// required for dynamic import using React.lazy()
+// eslint-disable-next-line import/no-default-export
+export default PrimaryActionPopover;
```

Comment:
A lot of these have the export default but are we using the React.lazy for these components? Do we still want to export them like this?

Reply:
Ah right, that's leftover from when I was trying to lazy load the components, but then Clint helped me realize that we probably shouldn't lazy load the toolbar. I'll remove the default export from all the solution toolbar components.